projects/plugins/boost/app/assets/src/js/features/module/module.tsx

```diff
@@ -7,6 +7,7 @@ type ModuleProps = {
 	title: React.ReactNode;
 	description: React.ReactNode;
 	children?: React.ReactNode;
+	alwaysRenderableChildren?: React.ReactNode;
```

Comment:
This needs a comment describing what it means, and what it's for. It's a confusing name. 

But at the same time, I struggle to come up with a better one. :-) So I recommend a comment.

Reply:
I've removed this since the notice is now going to be a child that will be visible only when toggled.